lib/plugins/aws/package/compile/functions/index.js

```diff
@@ -398,7 +398,7 @@ class AwsCompileFunctions {
 
       if (functionObject.reservedConcurrency || functionObject.reservedConcurrency === 0) {
         // Try convert reservedConcurrency to integer
-        const reservedConcurrency = _.parseInt(functionObject.reservedConcurrency);
+        const reservedConcurrency = Number(functionObject.reservedConcurrency);
```

Comment:
Just for know, Why not to use `Number.parseInt(x, base)`

https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/parseInt

Reply:
The only difference I could find about `Number()` and `Number.parseInt` is that `Number.parseInt(null)` returns `NaN` and `Number(null)` returns 0. So, I think it's safe to use `Number.parseInt`. Thanks @safv12 for pointing it out.

Reply:
`parseInt` is for parsing numbers out of strings. Technically logic is as:

```
resolveIntegerPrefixOutOfString(coerceToString(value))
```

While what `Number` does is:

```
coerceToNumber(value)
```

Here we expect user to input number type directly (and not string with integer prefix), so using `parseInt` seems as not perfectly justified and it'll also hide configuration error if someone puts `1.5`

As a side note input validation will shortly be handled by JSON schema based validation (see https://github.com/serverless/serverless/issues/6562) if we would have that already in, then best would be to just assume that `functionObject.reservedConcurrency` is good as is (as it's already validated), but until is the case, let's use `Number`